domFoundations/events,storage,attributes/buttonListener.js

```diff
@@ -0,0 +1,9 @@
+const swBtn = document.getElementById('swichButtn');
+
+export const buttonListener = (...events) => {
```

Comment:
probably the correct name of this variable is `handlers`. But for me is better to avoid a lot of handlers - you can use one with all operations inside it

Reply:
but it's up to you.

Reply:
Name - changed

Reply:
Please explain what do you mean by 'use one with all operations inside it'. If I make a single function to perform 4 different actions that are currently devided between the existing 4 handlers - wouldn't it be bad? In such case the new single function will be big and will perform more than one kind of action.